tests/distributed/test_basic_distributed_correctness.py

```diff
@@ -22,15 +22,17 @@
 @pytest.mark.skipif(cuda_device_count_stateless() < 2,
                     reason="Need at least 2 GPUs to run the test.")
 @pytest.mark.parametrize(
-    "model, distributed_executor_backend, attention_backend, test_suite", [
-        ("facebook/opt-125m", "ray", "", "L4"),
-        ("facebook/opt-125m", "mp", "", "L4"),
-        ("meta-llama/Llama-2-7b-hf", "ray", "", "L4"),
-        ("meta-llama/Llama-2-7b-hf", "mp", "", "L4"),
-        ("facebook/opt-125m", "ray", "", "A100"),
-        ("facebook/opt-125m", "mp", "", "A100"),
-        ("facebook/opt-125m", "mp", "FLASHINFER", "A100"),
-        ("meta-llama/Meta-Llama-3-8B", "ray", "FLASHINFER", "A100"),
+    "model, distributed_executor_backend, attention_backend, "
+    "test_suite, enable_adag", [
+        ("facebook/opt-125m", "ray", "", "L4", False),
+        ("facebook/opt-125m", "ray", "", "L4", True),
+        ("facebook/opt-125m", "mp", "", "L4", False),
+        ("meta-llama/Llama-2-7b-hf", "ray", "", "L4", False),
+        ("meta-llama/Llama-2-7b-hf", "mp", "", "L4", False),
+        ("facebook/opt-125m", "ray", "", "A100", False),
+        ("facebook/opt-125m", "mp", "", "A100", False),
+        ("facebook/opt-125m", "mp", "FLASHINFER", "A100", False),
+        ("meta-llama/Meta-Llama-3-8B", "ray", "FLASHINFER", "A100", False),
```

Comment:
for this, my suggestion is, don't surface `enable_adag` to the argument list, as this is ray specific.

Reply:
what about enable_spmd? I wanted to run both ("facebook/opt-125m", "ray", "", "L4") and (("facebook/opt-125m", "ray", "", "L4") + spmd (otherwise we regress the test coverage), so I don't want to check "if arg matches, enable spmd".

Reply:
renamed to enable_spmd